src/content/service.js

```diff
@@ -185,20 +186,23 @@ module.exports = async ({ botfile, projectLocation, logger }) => {
         .then()
     } else {
       const randomId = getNewItemId(category)
-      await knex('content_items')
-        .insert({
-          ...body,
-          createdBy: 'admin',
-          createdOn: helpers(knex).date.now(),
-          id: randomId,
-          categoryId
-        })
-        .then()
+      await knex('content_items').insert({
+        ...body,
+        createdBy: 'admin',
+        createdOn: helpers(knex).date.now(),
+        id: randomId,
+        categoryId
+      })
     }
 
     return dumpDataToFile(categoryId)
   }
 
+  const categoryItemsCount = async () => {
+    const [result] = await knex('content_items').count('id')
```

Comment:
I think naming the `count` is safer and cleaner

Reply:
you can also do `knex('content_items').count('id as count').get(0).get('count)`

Reply:
Yes, looks good, thanks! :+1: